input/fsh/moped-encounter.fsh

```diff
@@ -16,24 +16,22 @@ Title: "MOPED Encounter"
 * identifier[DatensatzID].type from http://terminology.hl7.org/ValueSet/v2-0203 (required)
 * identifier[DatensatzID].type.coding.code = #ANON (exactly)
 * identifier[DatensatzID] ^short = "SHA-256 verschlüsselte Aufnahmezahl"
-* account only Reference(MopedAccount)
-* account 0..1
```

Comment:
Frage: sollen wir Account beibehalten um die Coverage auch über den Encounter referenzieren können? Der hängt sonst kontextlos in der Luft im "normalen" Ressourcen-Netzwerk und ist die einzige Ressource, die nur über die Composition verknüpft ist.

Reply:
+1